.jshintrc

```diff
@@ -27,7 +27,7 @@
     "unused"          : true,      // Warns when you define and never use your variables.
     "strict"          : false,     // Requires all functions to run in ECMAScript 5's strict mode.
     "trailing"        : true,      // Makes it an error to leave a trailing whitespace in your code.
-    "maxlen"          : 120,       // Lets you set the maximum length of a line.
+    //"maxlen"        : 120,       // Lets you set the maximum length of a line.
```

Comment:
Other places seemed to comment out options so that's what I did, but in this case we will never want to re-enable it so I agree that it makes more sense to remove it. I've done so, squashed and pushed it up.